src/test/groovy/nebula/test/SpecifiedGradleVersionIntegrationSpec.groovy

```diff
@@ -23,7 +24,12 @@ class SpecifiedGradleVersionIntegrationSpec extends IntegrationSpec {
         when:
             def result = runTasksSuccessfully('build')
         then:
-            result.standardOutput.contains("gradle/$requestedGradleVersion/taskArtifacts")
+            if (Os.isFamily(Os.FAMILY_WINDOWS)){
```

Comment:
oh i know the ignoreif annotation, i just hate the dup code is all.  but if you want two tests, no problem, i can do that.  And as i write this, i just realized how to pull it off.  give me a few minutes